src/components/contracts/ExtendContractForm.vue

```diff
@@ -0,0 +1,360 @@
+        },
+        carryoverTargetDate: startDate,
+        carryoverMinutes: 0,
+        worktime: null
+      });
+    },
+    startDate: {
+      get() {
+        return localizedFormat(parseISO(this.contract.start), "yyyy-MM-dd");
+      },
+      set(val) {
+        const [year, month, day] = val.split("-");
+
+        const date = new Date(year, month - 1, day, 0, 0);
+        this.contract.start = localizedFormat(date, "yyyy-MM-dd");
+
+        // Update the carryoverTargetDate to the contract start, if we do not
+        // want to carry over anything
+        if (!this.carryover) {
+          this.contract.carryoverTargetDate = format(
+            startOfMonth(parseISO(this.contract.start)),
+            "yyyy-MM-dd"
+          );
+        }
+
+        // Update the carryoverTargetDate if we move the start month beyond the
+        // previous value
+        if (isAfter(date, parseISO(this.contract.carryoverTargetDate))) {
+          this.contract.carryoverTargetDate = format(date, "yyyy-MM-dd");
+        }
+
+        if (isAfter(date, parseISO(this.contract.end))) {
+          this.contract.end = localizedFormat(endOfMonth(date), "yyyy-MM-dd");
+        }
+      }
+    },
+    endDate: {
+      get() {
+        return localizedFormat(parseISO(this.contract.end), "yyyy-MM-dd");
+      },
+      set(val) {
+        const [year, month, day] = val.split("-");
+        this.contract.end = localizedFormat(
+          new Date(year, month - 1, day, 0, 0),
+          "yyyy-MM-dd"
+        );
+      }
+    },
+    nameErrors() {
+      const errors = [];
+      if (!this.$v.contract.name.$dirty) return errors;
+      !this.$v.contract.name.minLength &&
+        errors.push(
+          this.$t("errors.minLength", {
+            name: this.$t("errors.contractName"),
+            length: 2
+          })
+        );
+      !this.$v.contract.name.maxLength &&
+        errors.push(
+          this.$t("errors.maxLength", {
+            name: this.$t("errors.contractName"),
+            length: 100
+          })
+        );
+
+      !this.$v.contract.name.required &&
+        errors.push(
+          this.$tc("errors.nameRequired", 1, {
+            name: this.$t("errors.contractName")
+          })
+        );
+      return errors;
+    },
+    worktimeErrors() {
+      const errors = [];
+      if (!this.$v.contract.worktime.$dirty) return errors;
+      !this.$v.contract.worktime.required &&
+        errors.push(
+          this.$tc("errors.nameRequired", 1, {
+            name: this.$t("errors.hours")
+          })
+        );
+      !this.$v.contract.worktime.validWorktime &&
+        errors.push(this.$t("errors.timeFormat"));
+      !this.$v.contract.worktime.minLength &&
+        errors.push(this.$t("errors.timeFormat"));
+      !this.$v.contract.worktime.worktimeNotZero &&
+        errors.push(
+          this.$t("errors.durationBiggerZero", {
+            entity: this.$tc("models.contract")
```

Comment:
Der Eintrag fehlt wohl noch in der Übersetzung

Reply:
Also - tatsächlich wird auf diesen Eintrag auch beim Titel des Formulars zurückgegriffen (`FormDialog` &rarr; `entity` aufgerufen aus `ViewContractList`)